cmd/minikube/cmd/start.go

```diff
@@ -486,6 +497,16 @@ func generateConfig(cmd *cobra.Command, k8sVersion string) (cfg.Config, error) {
 	return cfg, nil
 }
 
+// noneNeedsResolveConf checks extra config option for resolv.conf is needed
+// for none driver. more info https://github.com/kubernetes/minikube/issues/3511
+func noneNeedsResolveConf() bool {
+	filename := "/run/systemd/resolve/resolv.conf"
+	if _, err := os.Stat(filename); os.IsNotExist(err) {
```

Comment:
optional: one could argue if there is any error calling os.Stat(), that we shouldn't use this file. I think you can safely remove the os.IsNotExist check.

Alternatively, you could also check for other more exceptional errors and log them.